tests/playwright/src/model/pages/kubernetes-dashboard-page.ts

```
@@ -48,7 +69,35 @@ export class KubernetesDashboardPage extends MainPage {
   }
 
+  async getCurrentTotalCountForResource(name: KubernetesResources): Promise<number> {
+    const currentResource = this.metricsLocatorByName(name);
+    const totalCountLocator = currentResource.getByLabel(`${name} count`, { exact: true });
+
+    if ((await totalCountLocator.count()) === 0) {
+      throw new Error(`No total count locator found for resource: ${name}`);
+    }
+
+    const totalCountText = await totalCountLocator.textContent();
+    return totalCountText ? parseInt(totalCountText, 10) : 0;
```

Biome (2.1.2)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)